### PR TITLE
fix(a11y): removed redundant a11y code from Dropdown

### PR DESCRIPTION
**Pull request by silviuaavram**
Cleanup getA11yStatusMessage and A11yStatusMessageOptions once updated versions get released in Downshift.

Now these messages are generated by default as a result of https://github.com/paypal/downshift/pull/629

### CODE REVIEW

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -71,7 +66,7 @@ export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownS
    * Callback that creates custom accessability message for dropdown status change. Involves changes in highlighted item in the list, selection, toggle status.
    * @param {A11yStatusMessageOptions<ShorthandValue>} messageGenerationProps - Object with properties to generate message from. See getA11yStatusMessage from Downshift repo.
```
Comment (layershifter) [inline src/components/Dropdown/Dropdown.tsx]: Oops, `@param` should be fixed, too :)

Comment (silviuaavram) [reply src/components/Dropdown/Dropdown.tsx]: fixed, thanks!

**Review by layershifter — APPROVED** [review]

**Review by mnajdova — APPROVED** [review]
Please update the changelog before merging.

**Comment by jurokapsiar** [comment]
nice cleanup. please make sure to have at least one example that shows how the messages can be provided by the consumer for translation purposes.

**Comment by silviuaavram** [comment]
@jurokapsiar I've added an example in French. I wanted to be clear why those methods are needed for (internationalisation).

**Comment by jurokapsiar** [comment]
Cette liste déroulante est fabuleuse!

**Pull request merged**